Solutions/moritz_w/Lesson_6/integer_sort.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+
+def insertion_sort(data):
+    sorted_data = [i for i in data]
```

Comment:
You can copy list using this:
sorted_data = data[:]
Remember that it copies only list of references. 
To copy entire object you have to use "copy" module.

But for sorting copy of  list of references is enough.